book/content/remote_collaboration/checklist/checklist.md

```diff
@@ -798,30 +798,58 @@ This checklist also aims to help participants support the event organisers by be
 
 <a name="checklist-for-managing-and-participating-in-a-distributed-team"></a>
 ## Checklist for managing and participating in a distributed team
-*HELP WANTED*
+Managing and participating in a distributed team can be difficult. This is the magic spell for increasing productivity and team happiness for a distributed team. 
 
 <a name="tips-for-all-the-members"></a>
 ### Tips for all the members
-<details>
-  <summary>Click to expand</summary>
-
-</details>
+- [ ] Be courteous and have realistic exceptations from your colleagues  
+- [ ] Complete the tasks assigned to you
+- [ ] Always speak up your doubt to have a clear perspective on the team goals
+- [ ] If not able to attend a corresponding online meeting, send regards prior to the manager
```

Comment:
Is 'send regards' really what is meant here? In my mind (maybe I'm wrong) 'send regards' is equivalent to 'say hi' -- and I guess you want to let the meeting organiser know you will not be able to make it.

Proposed rewording:
If not able to attend an online meeting, send your apologies to the meeting organiser

Reply:
Yeah, my mistake. Correcting it.